Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -377,6 +377,11 @@ synchronized public static Branch getAutoInstance(@NonNull Context context) {
             // Should only be set in json config
             deferInitForPluginRuntime(BranchUtil.getDeferInitForPluginRuntimeConfig(context));
 
+            String apiUrl = BranchJsonConfig.getInstance(context).getAPIUrl();
+            if (!TextUtils.isEmpty(apiUrl)) {
+                setAPIUrl(apiUrl + "/");
```

Comment:
So ios takes with no / but android does, but the api will take care of it. Maybe add a comment, or note in public documentation that custom api urls should not contain a trailing /